x-pack/plugin/eql/src/test/resources/queryfolder_tests.txt

```diff
@@ -10,6 +10,8 @@
 // ...
 // <expectation n>
 // ;
+// NOTE: do not use whitespaces in the relevant parts of the query itself, as they will be removed. For example 'process where process_name : "System Idle Process"'
```

Comment:
I think it's better to have a FIXME here?

Reply:
@matriv the code works as intended. It removes whitespaces without discrimination from query string so that it's easier to compare that string with what's in the text file. I put NOTE because it came as a surprise to me this side effect and had to adjust my test.